### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnfraney**
Add Emmet integration instructions

### CODE REVIEW

**Comment by djch** [comment]
Yeah, right on :+1:

File: README.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ It's common for Django templates to have a `.html` extension. Unfortunately, thi
 
 ![Settings Example](https://f.cloud.github.com/assets/69549/2429497/3fd16d50-ac86-11e3-9b78-7c2bc051212c.png)
 
+## Emmet Integration
+Out of the box, Emmet isn't aware of the Django HTML Templates grammar, so Django HTML Templates uses Atom's tab-based HTML autocompletion rather than Emmet's. As the [Emmet-Atom Readme](https://github.com/emmetio/emmet-atom) explains, this is an easy fix. All you have to do is add Djanto Templates to the the list of grammars in Emmet's `keymaps/emmet.cson` file:
```
Comment (djch) [inline README.md]: You've got a "Djanto" in here, by the way.

**Comment by johnfraney** [comment]
I made a small update. I noticed that Emmet was in XML mode (or something of the sort) for Django templates, so I tweaked the Django template grammar in Emmet, and now things are working properly.

**Comment by benjohnson** [comment]
Thanks all!

**Pull request merged**